src/h5web/visualizations/shared/utils.ts

```diff
@@ -61,42 +59,14 @@ export function isIndexAxisConfig(
   return 'indexDomain' in config;
 }
 
-export function getDataScaleFn(isLog: boolean): DataScaleFn {
-  return isLog ? scaleSymlog : scaleLinear;
-}
-
-function getAxisScale(config: AxisConfig, rangeSize: number): AxisScale {
```

Comment:
This function drops in complexity because we've already computed the scale function, and the domain can be accessed from the same `domain` key.